features/invitelist/impl/src/main/kotlin/io/element/android/features/invitelist/impl/InviteListPresenter.kt

```diff
@@ -51,10 +53,19 @@ class InviteListPresenter @Inject constructor(
             .roomSummaries()
             .collectAsState()
 
-        val seenInvites = remember { store.seenRoomIds }
+        val seenInvites = remember { mutableStateOf<Set<RoomId>>(emptySet()) }
```

Comment:
Why not using the `collectAsState` ?

Reply:
We don't want the latest value, but the value as of when the invite list was opened.

e.g.:

- the seen invite list is `[roomA]`
- the user is invited to `roomB`
- the user navigates to the invite list
- the seen invite list is updated to `[roomA, roomB]`
- but `seenInvites` stays as just `[roomA]` so `roomB` is shown as new until the user navigates away